src/components/AddBook/index.js

```diff
@@ -185,6 +150,9 @@ function AddBook() {
         <p>{titleErrors}</p>
         <p>{authorErrors}</p>
         <p>{genreErrors}</p>
```

Comment:
Just noticed that the error you get is 'genre_id field is required' - that's going to confuse the hell out of a user. Because the API is dealing with IDs, but the user isn't.